packages/connect-popup/e2e/tests/permissions.test.ts

```diff
@@ -18,23 +19,49 @@ const fixtures = [
     {
         description: `iframe and host same origins but with trust-issues=true -> show permissions`,
         queryString: '?trust-issues=true',
+        setTrustedHost: false,
```

Comment:
if we would ensure possibility to downgrade trust level in TrezorConnect.init parameter, do we still have a use case for trust-issues query string param? 

on the other hand - we can pass connectSrc both in init and query string 🤔

Reply:
Yes, that is a good question I also have.
Should we have both ways 
1. set by query param
2. set by connect-explorer new setting page
:question: 

And I think if the reason for using query param `trust-issues` is only for testing I would get rid of that and use only the connect-explorer settings page.

Reply:
1. trust-issues is only for testing
2. if we can have TrezorConnect.init({ trustedHost: false }) (and false is the only accepted option), then it could do the same. 
3. query string is little bit more convenient, on the other hand, it is not possible to use it in webextensions anyway

So yes, I am slightly in favor of removing trust-issues and implementing its functionality in trezorconnect.init